dkg-runtime-primitives/src/handlers/decode_proposals.rs

```diff
@@ -67,7 +75,9 @@ pub fn decode_proposal<C: ChainIdTrait>(
 			ChainIdType::EVM(_) => evm::fee_update::create(&proposal.data()).map(|p| {
 				(p.header.chain_id, DKGPayloadKey::WrappingFeeUpdateProposal(p.header.nonce))
 			}),
-			ChainIdType::Substrate(_) => todo!(),
+			ChainIdType::Substrate(_) => substrate::fee_update::create(&proposal.data()).map(|p| {
```

Comment:
We can do a triple match here since these proposals will work on what we designate as Parachains and Relaychains.
```
ChainIdType::Substrate(_) | ChainIdType::RelayChain(_,_) | ChainIdType::Parachain(_)
```

Reply:
fixed